api/specs.proto

```diff
@@ -103,6 +103,7 @@ message GlobalService {
 
 message TaskSpec {
 	oneof runtime {
+		NetworkAttacherSpec attacher = 8;
```

Comment:
@dongluochen This is the new runtime we are creating for attaching docker unmanaged containers(created using `docker run`). In this runtime we don't actually manage the container itself but just the attachment.